epitator/count_annotator_alt.py.bak

```diff
@@ -0,0 +1,188 @@
+#!/usr/bin/env python
```

Comment:
Could you remove this and the other unused files. It would make the PRs easier to review. The code will still be preserved in the commit history.

Reply:
Oops, my mistake — I thought I had removed them.